main.go

```diff
@@ -22,6 +22,8 @@ import (
 	"log"
 	"log/syslog"
 	"net"
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
Force-imports the package even if it's not used directly. Necessary in order to get the side-effect of having net/http/pprof's HTTP handlers installed.